packages/core/src/growing-entry/growing-entry.tsx

```diff
@@ -23,6 +23,8 @@ export const GrowingEntry: React.FunctionComponent<Props> = (props: Props) => {
 
     assert(onChange !== undefined, "GrowingEntry must be a controlled input area");
 
+    const [inputID] = React.useState(() => "input-box-" + Math.round(Math.random() * 1000));
```

Comment:
This should probably use something like `useId()` instead https://react.dev/reference/react/useId

Reply:
That was my first thought to use `useId()`, but I decided to make it similar to [`searchID`](https://github.com/quicktype/glide-data-grid/blob/c9e1f3df5d545171ccca2226c0c55e253fb6cc83/packages/core/src/data-grid-search/data-grid-search.tsx#L78).
Would you like me to update it with `useId`?

Reply:
Oh hrm, I feel like that should be using `useId()` as well, but honestly not really sure if there's a good reason to / not to. In general I feel like it's better to avoid peppering `Math.round(Math.random() * 1000)`-type things around though.

Reply:
Oh, it appears that there's no `useId` in `"react": "^16.12.0 || 17.x"` so we can't use it, unfortunately.